packages/docs/docusaurus.config.js

```diff
@@ -74,18 +74,29 @@ const config = {
                         type: 'doc',
                         docId: 'welcome',
                         position: 'left',
-                        label: 'API',
+                        label: 'Docs',
```

Comment:
I'd prefer to keep this `API`. I usually have to search doc sites for the `API` link to find the list of modules/functions.

Reply:
I will vote for Docs or Documentation,
most of other doc are using this term.

also the "API" is not just contains APIs, there is also other pages, like welcome..etc.